This patch addressed bug# 15132887
GNU Ghostscript should not enable decryption of PDFs when shipped with Solaris

Note : Not suitable for fixing upstream, as this Solaris specific.

--- ghostscript-9.22/Resource/Init/pdf_sec.ps.orig
+++ ghostscript-9.22/Resource/Init/pdf_sec.ps
@@ -472,12 +472,12 @@
     } ifelse
   } ifelse
 
-%   Trailer /Encrypt oget /P oget 4 and 0 eq #? and
-%    { (   ****This owner of this file has requested you do not print it.\n)
-%      pdfformaterror printProducer
-%      /pdf_process_Encrypt cvx /invalidfileaccess signalerror
-%    }
-%   if
+  Trailer /Encrypt oget /P oget 4 and 0 eq #? and
+   { (   ****This owner of this file has requested you do not print it.\n)
+     pdfformaterror printProducer
+     /pdf_process_Encrypt cvx /invalidfileaccess signalerror
+   }
+  if
 } bind def
 
 % Calculate the key used to decrypt an object (to pass to .decpdfrun or
